install.ps1

```diff
@@ -202,7 +202,7 @@ if (-not $noChecks.IsPresent) {
     # Prompt user to remind them to take a snapshot
     Write-Host "[-] Do you need to take a VM snapshot before continuing? (Y/N): " -ForegroundColor Yellow -NoNewline
     $response = Read-Host
-    if ($response -notin @("n","N")) {
+    if ($response -in @("y","Y")) {
```

Comment:
I meant to phrase the question so the answer is `Y` and it continues :)

Reply:
Ahhhh, 🤦

Reply:
Updated the question, let me know if you think it sounds okay

Reply:
That's good! Thanks.